recipes.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Recipes</title>
+</head>
+<body>
+  <p>
+    <h2>Recipes</h2> ...and tasty bites from <em>all around the world.</em>
```

Comment:
since h2 is a block element, you might consider doing a new line for the extra text